tests/cephfs/cephfs_utilsV1.py

```diff
@@ -1718,79 +1723,64 @@ def cephfs_nfs_mount(self, client, nfs_server, nfs_export, nfs_mount_dir):
         return rc
 
     @staticmethod
-    def nfs_ganesha_conf(node, nfs_client_name):
-        out, rc = node.exec_command(
```

Comment:
Why this method is removed?
I think this method is used in upgrade test file

Reply:
I replaced my cephfs_utils file so it would have gone due to that.
It's fixed now.